### DISCUSSION TITLE
Possible Change Needed in LogEntryEventHandler.cls > setStatusApiDetails

### DISCUSSION THREAD

**Discussion by shuylern** [jongpie/NebulaLogger#283] [General]
Received this Error recently: 
_Failed to invoke future method 'private static void setStatusApiDetails()' on class 'LogEntryEventHandler' for job id '7075f00001GwEml'_

I believe the method utilized has been deprecated.

![image](https://user-images.githubusercontent.com/64104923/156777206-2330e6e7-d21a-4131-a2c2-5bd0800ed311.png)

**Comment by jongpie** [comment]
Hi @shuylern - do you have any more details on the exception that occurs when trying to run the method `setStatusApiDetails()`? I'm not able to reproduce the issue in any of my scratch orgs/sandboxes. I also took a look at the API docs, and the endpoint in your screenshot is not the same endpoint being used by Nebula Logger:

- Your screenshot shows the deprecated endpoint `/instances/status` (red box in screenshot below) - this is not exactly the endpoint currently being used
- Nebula Logger is currently using the endpoint `/instances/{key}/status` (green box in screenshot below), which has not been deprecated 

![image](https://user-images.githubusercontent.com/1267157/156969216-9f9c2113-a78c-43ce-b6db-7595df285a64.png)

**Reply by shuylern** [reply]
@jongpie,

Full error below in case it helps.  

Does appear to be my mis-read of the URL.  

Thanks!  

===============================
_Apex script unhandled exception by user/organization: 0055f000008****/00D5f000009****

Failed to invoke future method 'private static void setStatusApiDetails()' on class 'LogEntryEventHandler' for job id '7075f00001FNCzJ'

caused by: System.CalloutException: Unauthorized endpoint, please check Setup->Security->Remote site settings. endpoint = https://api.status.salesforce.com/v1/instances/*****/status

Class.LogEntryEventHandler.setStatusApiDetails: line 428, column 1_

**Reply by jongpie** [reply]
@shuylern it sounds like there's an issue with the remote site setting in your org - Nebula Logger includes a remote site setting for the site https://api.status.salesforce.com, and it sounds like it's either been disabled or deleted in your org. Can you check your org under Setup --> Remote Site Settings? You should have a remote site setting like the one in the screenshot below - if it's missing, you'll need to manually create it. If it's present, but inactive, then you'll need to activate it.

![image](https://user-images.githubusercontent.com/1267157/157050572-d8c7d981-f6f3-47a0-a8ba-303c47d8b82c.png)

**Reply by shuylern** [reply]
Thanks for the explanation @jongpie . That allowed me to make the necessary updates.
